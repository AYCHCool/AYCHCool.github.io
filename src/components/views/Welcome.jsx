import React from 'react';

const Welcome = () => (
  <div>
    <h1>Hello Mazus</h1>
  </div>
);

export default Welcome;
