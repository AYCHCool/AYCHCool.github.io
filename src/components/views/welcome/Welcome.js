import React from 'react';
import { Link } from 'react-router-dom';

const slug = 'building-apis-with-nodejs-48458493';
const Welcome = () => (
  <div>
    <h1>Hello Mazus</h1>
    <Link to={`/article/${slug}`}>View Sample Article</Link>
  </div>
);

export default Welcome;
