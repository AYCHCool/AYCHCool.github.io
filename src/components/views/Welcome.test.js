import React from 'react';
import Welcome from './Welcome';

it('renders correctly', () => {
  const wrapper = mount(<Welcome />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
