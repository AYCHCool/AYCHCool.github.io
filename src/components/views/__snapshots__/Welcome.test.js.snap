// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Welcome>
  <div>
    <h1>
      Hello Mazus
    </h1>
  </div>
</Welcome>
`;
